sonnenbatterie/sonnenbatterie.py

```diff
@@ -4,8 +4,9 @@
 # pylint: disable=unused-wildcard-import
 from .const import *
 # pylint: enable=unused-wildcard-import
+from .timeofuse import timeofuseschedule
 
-
+print("Loading sonnenBatterie")
```

Comment:
i think this print statement is executed everytime the module is loaded. Is this good practice in python?
(i am coming from dotnet...)

Reply:
I have no idea, it was a debug thing for me, I'll remove it and recommit